src/main/java/com/cognifide/aemrules/extensions/CheckListRegistrar.java

```diff
@@ -50,25 +51,26 @@
 
   public static final String REPOSITORY_KEY = "AEM Rules";
 
-  public static final List<Class<? extends JavaCheck>> CHECK_CLASSES
-      = ImmutableList.<Class<? extends JavaCheck>>builder()
-      .add(AdministrativeAccessUsageCheck.class)
-      .add(AnnotationsConstantsCheck.class)
-      .add(ConstantsCheck.class)
-      .add(HttpConstantCheck.class)
-      .add(InjectorShouldBeClosedCheck.class)
-      .add(InjectorTryWithResourcesCheck.class)
-      .add(ModelsShouldNotUseSessionCheck.class)
-      .add(IteratingResourcesCheck.class)
-      .add(JcrPropertyFieldsInConstructorCheck.class)
-      .add(PreferSlingServletAnnotation.class)
-      .add(ResourceResolverShouldBeClosed.class)
-      .add(SessionShouldBeLoggedOut.class)
-      .add(SynchronizedKeywordUsageCheck.class)
-      .add(ThreadSafeFieldCheck.class)
-      .add(DefaultInjectionStrategyAnnotationCheck.class)
-      .add(ModifiableValueMapUsageCheck.class)
-      .build();
+	public static final List<Class<? extends JavaCheck>> CHECK_CLASSES
+		= ImmutableList.<Class<? extends JavaCheck>>builder()
```

Comment:
How about 

    ImmutableList.of(
         AdministrativeAccessUsageCheck.class,
         // ...
         ContentResourceShouldBeNullCheckedCheck.class
      );

Reply:
done